README.md

```diff
@@ -41,7 +41,8 @@ Before you get started with the authoring work, it's necessary that you understa
 * **[`docs-staging`](https://github.com/zowe/docs-site/tree/docs-staging/docs)** - protected branch
 
   Docs for the upcoming `vNext` release. When Zowe has a release, its `docs-staging` branch will be merged into `master` and the content will be visible on [https://docs.zowe.org/stable](https://docs.zowe.org/stable).
-* **[`active-development`](https://github.com/zowe/docs-site/tree/active-development/docs)** - protected branch
+
+* (REMOVED TEMPORATRILY UNTIL NEW COMPONENT FEATURES ARE AVAILABLE) **[`active-development`](https://github.com/zowe/docs-site/tree/active-development/docs)** - protected branch
```

Comment:
"TEMPORATRILY" -> "TEMPORARILY"

Reply:
Thank you, as you can see I have not tried the spell checker VSCode plugin 😝